src/android/Bugsnag.cpp

```diff
@@ -229,35 +222,40 @@ JNIEXPORT void JNICALL Java_com_bugsnag_android_BugsnagCocos2dxPlugin_configureN
       bugsnag::HashMapClass, "put",
       "(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;");
 
+  bugsnag::BreadcrumbTypeClass = env->FindClass("com/bugsnag/android/BreadcrumbType");
+  _mapBreadcrumbType(env, bugsnag::Navigation, "ERROR");
```

Comment:
```suggestion
  _mapBreadcrumbType(env, bugsnag::Navigation, "NAVIGATION");
```

Reply:
Damn... I'll open a new PR.

Reply:
no worries, it took me a while to finish looking at this one!